Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM jlesage/baseimage-gui:alpine-3.21-v4
+FROM jlesage/baseimage-gui:ubuntu-24.04-v4
 
 COPY --chmod=775 startapp.sh /startapp.sh
+ADD https://raw.githubusercontent.com/nextcloud/vm/refs/heads/main/static/fetch_lib.sh /var/scripts/fetch_lib.sh
+ADD https://raw.githubusercontent.com/nextcloud/vm/main/lib.sh /var/scripts/lib.sh
+ADD https://raw.githubusercontent.com/nextcloud/vm/refs/heads/main/not-supported/smbserver.sh /smbserver.sh
```

Comment:
also need to modify the script to allow normal directories, not only mountpoints: https://github.com/nextcloud/vm/blob/b98a75bd503427366a566db699c09474196c7813/not-supported/smbserver.sh#L56-L60

Reply:
Also systemctl is not supported...

Reply:
I could write a patch and apply the patch here...